src/gui/properties/piecesbar.cpp

```diff
@@ -100,9 +100,9 @@ namespace
             m_stream << "</table>";
         }
 
-        void operator()(const QString &size, const QString &path)
+        void operator()(const QString &size, const Path &path)
```

Comment:
> You should probably recheck all .cpp files whether the Path header is included...

You're doing it better. Probably because it's not a problem for me to have indirect inclusions (at least for such global things).

Reply:
Included here.